content/how-to-todo-in-ember-template-lint.md

```diff
@@ -0,0 +1,121 @@
+
+### Why, though?
+
+This new feature is really intended to help apps at scale. It can be tricky to update a dependency like `ember-template-lint`, especially if it's a version that includes a new rule, due to the number of factors involved. Taking in a new version of the linter usually means a lot of communication and coordination about the new rule(s) and the larger your app gets, and the more developers you have, the more work this was. For global companies that use Ember, there might even be a committee whose sole focus is to make decisions about linting rules. That's a lot of red tape!
+
+While the `pending` feature gave us _some_ of the desired functionality, it didn't really ensure that teams would be given the support they needed to actually fix existing issues- issues could stay "pending" forever. Now that the `todo` feature has replaced the `pending` feature, we can be confident that when we introduce new linting rules, we're giving our development teams the support they need to remember to fix those issues.
+
+Perhaps a visualization would also be useful:
+<img src="/images/blog/how-to-todo-in-ember-template-lint/value-visualization.png" alt="a chart that shows how existing code receives a todo, while new code must follow the linting rules.">
+
+All new code is immediately supported by new linting rules; existing code can have a timeline to provide a fix.
+
+We can even go a step farther, and customize the number of days to fix based on the rule itself. Super useful!
+
+## Step 1: Strategize
+
+You'll want to think about how you want to approach all of the linting rules. Do you want all rules to be treated the same way, or do you want to apply decay days by rule? The linter gives you options for both.
+
+### All rules (generic setting)
+
+It's possible that you don't care what the error is; all todos should have 15 days before they turn into a warning, and 30 days before they turn into an error. To do this, add this to the `package.json` file:
+
+```json
+{
+  "lintTodo": {
+    "decayDays": {
+      "warn": 15,
+      "error": 30
+    }
+  }
+}
+```
+
+The default values will vary from team to team, but in general the idea is this: it should be reasonable enough for teams to methodically work on past issues, but not too long that they are forgotten about. _*It should feel like a little bit of a stretch, but not stressful.*_
+
+### Increasing specificity by rule
+
+It's also possible that you'd want to specify that a particular rule needs to be fix with more immediacy. In that case, you'd want to run that rule by itself. Let's use the `no-nested-interactive` rule as an example:
+
+```bash
+yarn ember-template-lint app/ --no-config-path --no-inline-config --rule 'no-nested-interactive:error' --update-todo --todo-days-to-warn=7 --todo-days-to-error=14
```

Comment:
Is it possible to break this long line into multiple lines and have the lines run in a terminal? Something like:

```suggestion
yarn ember-template-lint app/ \
  --no-config-path \
  --no-inline-config \
  --rule 'no-nested-interactive:error' \
  --update-todo \
  --todo-days-to-warn=7 \
  --todo-days-to-error=14
```

I haven't tried this so I'm not sure if `\` is the correct syntax for breaking down a line.

Reply:
I have no idea. I think it's an atypical way to run a command, so I'd probably prefer to leave it in one line.

Reply:
Yep, 1 line sounds good!